recipe/meta.yaml

```diff
@@ -1,38 +1,35 @@
 {% set name = "pybamm" %}
 {% set version = "24.9.0" %}
+{% set python_min = "3.9" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 743dac0d17bd2f773050fe841cf3581658283f886e27795d9f5fd67d1ff563b5
 
 build:
-  number: 1
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
-  skip: true  # [py<39 or py>312]
```

Comment:
We would still need this skip because we don't support Python 3.13 yet. Can we also add a `python_max` variable at the top and use it?

An update to support Python 3.13 for the next release (soon) is in progress, though: https://github.com/pybamm-team/PyBaMM/pull/4552 – linked it here for completeness.

Reply:
Done!